src/python/pants/backend/awslambda/python/target_types.py

```diff
@@ -58,6 +61,26 @@ class ResolvePythonAwsHandlerRequest:
     field: PythonAwsLambdaHandlerField
 
 
+class PythonAwsLambdaLayout(Enum):
+    LAMDBA = "lambda"
```

Comment:
Note to self: this would probably be less confusing and better align with AWS terminology as:

```suggestion
    FUNCTION = "function"
```

Reply:
Yeah, that terminology fix makes sense.